imagetest/test_suites/network/setup.go

```diff
@@ -73,7 +73,17 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 		return err
 	}
 
+	vm3, err := t.CreateTestVM("vm3")
+	if err != nil {
+		return err
+	}
+	vm3.ChangeNicTypeToGVNIC()
+	if err := vm3.Reboot(); err != nil {
```

Comment:
what is the reason to have VM 3 reboot?

Reply:
I just followed the previous approach used for VM 2 wherein after adding the AliasIpRanges to the network interface the VM is rebooted.

Reply:
the reboot is necessary for the test in that case - we have a test to ensure the alias IP ranges survive a reboot.